vagrant/provision.sh

```diff
@@ -31,7 +31,7 @@ chmod 777 /var/maestro
 echo "#!/bin/bash
 . /etc/profile.d/envvars.sh
 cd $MAESTRO_SRC
-$GOBIN/maestro
+exec $GOBIN/maestro
```

Comment:
Huh. Any reason why this needed the exec?

Reply:
@ryannowarm may be able to answer this (he was my go-to person to ask questions as linux management and programming is not my strongpoint)

I *believe* it was for sourcing the proper environment variables since it executes maestro in a new bash

Reply:
@cmonr , it's a performance hack.  exec replaces the current process(the bash script) with maestro since there's nothing the script needs to do after maestro is started.  Might as well get rid of that bash process since we don't need it at that point.  If it bothers you, we can get rid of it.  (No real harm in not using exec except extra process hanging around)

Reply:
If that's all, then I'm fine with it.

Although I'm curious how much performance is being recovered by the hack.

Reply:
@cmonr basically the process space(RAM, space in your ps -aef, etc) taken up by one extra bash process.